Mathlib/CategoryTheory/Category/Factorisation.lean

```diff
@@ -74,6 +76,71 @@ instance : Category.{max u v} (Factorisation f) where
   id d := Factorisation.Hom.id d
   comp f g := Factorisation.Hom.comp f g
 
+/- We now aim to show that `Factorisation f` is equivalent to iterated comma categories
+in two different ways.
+Namely, given `f : X ⟶ Y`, we will have `(X/C)/f` ≌ `Factorisation f` ≌ `f/(C/Y)`.
+-/
+section IteratedCommaCategories
+
+variable (f : X ⟶ Y)
+
+/- `Factorisation f ≌ (X/C)/f` -/
+section OverOfUnder
+
+/-- We aim to show `Factorisation f` ≌ `(X/C)/f`. That is to say,
+we aim to show the two functors we defined below are inverses of each other.
+-/
+def factorisationEquivOverUnderMk : Factorisation f ≌ Over (Under.mk f) where
```

Comment:
```suggestion
@[simps]
def factorisationEquivOverUnderMk : Factorisation f ≌ Over (Under.mk f) where
```
This attribute generates lemmas describing how the functor acts, you can see them by doing `simps?` instead